_meeting-notes/15-10-19.md

```diff
@@ -0,0 +1,115 @@
+  - Sync with Jim on TestGround plan and priorities https://github.com/ipfs/testground/pull/63
+  - Restructured how we run meetings and how we follow up to action items  https://github.com/ipfs/testground#team
+- ==@jimpick==
+  - (was sick much of last week)
+  - Sync with David on TestGround plan and priorities https://github.com/ipfs/testground/pull/63
+  - Elasticsearch ... using "Elasticsearch" service via AWS Marketplace
+    - Fairly generic Filebeat setup (JSON, Container input). Was missing data, fix is to increase polling frequency to 1s or tell testground to retain containers. https://gist.github.com/jimpick/fb3e5c841ae5d8c2436a5f8654bb0cf2
+    - [Demo video](https://ipfs.io/ipfs/Qmahm2x4UGqGgkeVhijTF1SvmekhiphfLgK2uQxTU95y8P/testground-filebeat.mp4) (23s)
+  - Hashicorp Packer
+    - Docker CE, go 1.13, Filebeat, and redis-server
+    - tested with EC2 auto-scaling group, Docker Swarm
+    - https://github.com/jimpick/testground-packer-templates
+  - Docs / Issues
+    - Document Cloud Setup on AWS https://github.com/ipfs/testground/pull/62
+      - Needs a bit more discussion
+      - Recorded a video showing how to setup admin account ... initially thought video would be easier, but going to switch to web tutorial due to need to blur things if sharing publicly
+      - Best way to communicate? Place to accumulate docs? (testground repo feels overloaded to me)
+    - Rationales / decision issues
+      - Not sure best place to have these conversations? GitHub might be the wrong place ... not intended to question spec or decisions that have been made
+      - [vs. Testlab](https://github.com/ipfs/testground/issues/64)
+      - [vs. ipfs/benchmarks](https://github.com/ipfs/testground/issues/66)
+      - [Elasticsearch Options](https://github.com/ipfs/testground/issues/67)
+      - [Grafana vs. Kibana](https://github.com/ipfs/testground/issues/68)
+  - Made "playground" repo for experimentation
+  - Booked tickets to Tenerife
+  
+
+## :question: Questions (Blockers, Technical Design needs)
+
+- (@daviddias) How should I go about tests that require ingesting multiple TBs of data? (non-blocker for this week)
+    - Options:
+      - Use EBS snapshot. Mount for the test
+      - Questions:
+        - Will we have infrastructure provider dependent tests?
+- (@daviddias) (might) Need a tip on getting smlbench to work https://github.com/ipfs/testground/pull/60/files#r331433948
+- (@jimpick) Improvements to communication / documentation - I'm trying to restrict myself to GitHub, but unsure of how to structure things...
+  - How to deal with Secrets?
+    - Use <INSERT TOKEN HERE> notes and then let people know (FAQ) that they need 1password
+  - All in one repo for now (docs, examples, tutorials, code)
+
+## :dart: Next
+
+### Top 3 things to get done this week
+
+- Ship Test Plan 1
+- Ship Test Plan 2
+- Get results of both Test Plans outputting to Elastic Search
+
+### Assignments
```

Comment:
I'll have all of these represented in trello by the end of the day today. Now back to the Research PM Summit